src/SharedPipelineComponents/DataLockEvents/SFA.DAS.Provider.Events.DataLock.IntegrationTests/SFA.DAS.Provider.Events.DataLock.IntegrationTests.csproj

```diff
@@ -145,6 +145,9 @@
       <Link>DbSetupScripts\MigrationScripts\004_DataLockEvents_Add_TransactionTypesFlag.sql</Link>
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\DeployDataLock\sql\migrationscripts\005_DataLockEvents_Tables_Add_Indexes.sql">
+      <Link>DbSetupScripts\MigrationScripts\005_DataLockEvents_Tables_Add_Indexes.sql</Link>
+    </Content>
```

Comment:
` <CopyToOutputDirectory>Always</CopyToOutputDirectory>`

Reply:
Change made and Data Lock and Data Lock Events integration tests successfully run.